Use guid instead of _id from cloud api and add paging to search results

hanatgit opened the pull request:
- use guid instead of _id (api change)
- add paging to search results
- show search term in result page

hanatgit committed:
Update Webonary_Cloud.php

use guid instead of _id

phillip-hopper reviewed (approved):
LGTM :+1:

hanatgit committed:
keep 'g' in guid

hanatgit committed:
keep 'g' in guid

hanatgit committed:
Update Webonary_Cloud.php

hanatgit committed:
add paging to search results for cloud api

also show search term

hanatgit committed:
add paging to search results for cloud api

also show search term

hanatgit committed:
spacing

hanatgit committed:
Update search.php

hanatgit committed:
Update search.php

hanatgit committed:
Update search.php

hanatgit committed:
Merge branch 'webonary-wordpress-develop' of https://github.com/sillsdev/webonary into webonary-wordpress-develop

phillip-hopper commented on wordpress/wp-content/themes/webonary-zeedisplay/search.php:
```diff
@@ -10,26 +10,37 @@ function openImage(image)
 		}
 		?>
-		<h2 class="arh"><?php printf( __('Search results for "%s"', ZEE_LANG), $searchquery);?></h2>
+		<h2 class="arh"><?php printf( __('Search results for "%s"', ZEE_LANG), $search_query);?></h2>
```
`$search_query` could be an empty string here (from line 32 or 37).

hanatgit replied:
Yes, $search_query will be an empty string if the user hits the search button without specifying anything in the search text box. We should probably catch that with javascript validation on the search form. I had initialized $search_query to an empty string to avoid PHP warning.  Since searching for an empty string will return an empty post, the complete message that will appear back to the user is as below, which should suffice in informing the user that they specified a search for nothing.
<img width="881" alt="Screen Shot 2020-06-14 at 9 37 53 PM" src="https://user-images.githubusercontent.com/60716623/84612898-97463480-ae87-11ea-979d-584ae7b65545.png">

phillip-hopper reviewed (changes requested):
Also, `echo _e('Comments', ZEE_LANG);` on line 68 is incorrect. The function `_e` doesn't return anything, it's a substitute for `echo`.

I know you didn't change that line.

hanatgit committed:
Update search.php

hanatgit commented:
> Also, `echo _e('Comments', ZEE_LANG);` on line 68 is incorrect. The function `_e` doesn't return anything, it's a substitute for `echo`.
> 
> I know you didn't change that line.

Changed to `<?php _e('Comments', ZEE_LANG); ?>`
The diff is hard to read because I fixed some spacing as well.

phillip-hopper reviewed (approved):
LGTM
